crates/symbolicator-js/src/symbolication.rs

```diff
@@ -154,7 +162,7 @@ async fn symbolicate_js_frame(
         .map(|entry| entry.sourcemap_url())
         .ok()
         .flatten()
-        .unwrap_or_else(|| raw_frame.abs_path.clone());
+        .unwrap_or_else(|| raw_frame.abs_path.clone().unwrap_or_default());
```

Comment:
Can use `abs_path.to_owned()` here now, I think, since we already checked for `Some` before.

```suggestion
        .unwrap_or_else(|| abs_path.to_owned());
```

Reply:
I had to revert this in [12d1d5e](https://github.com/getsentry/symbolicator/pull/1581/commits/12d1d5eb248c36d5d071f8a6a3f0c92699d30ea0) to fix a compilation error

```
error[E0502]: cannot borrow `*raw_frame` as mutable because it is also borrowed as immutable
   --> crates/symbolicator-js/src/symbolication.rs:145:17
    |
105 |   ...ath = raw_frame
    |  __________-
106 | | ...path
    | |_______- immutable borrow occurs here
...
145 |   ...   apply_source_context(raw_frame, minified_source.contents(...
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ mutable borrow occurs here
...
163 |   ...ap_or_else(|| abs_path.to_owned());
    |                    -------- immutable borrow later captured here by closure

```